i18n/hi/strings.json

```diff
@@ -135,7 +135,7 @@
   "editor.nav.tools.custom_domains": "कस्टम डोमेन्स",
   "editor.nav.tools.database": "डेटाबेस",
   "editor.nav.tools.plugins": "प्लगइन्स",
-  "editor.nav.tools.scheduled_jobs": "अनुसूचित कार्य",
+  "editor.nav.tools.scheduled_jobs": "Cron Jobs",
```

Comment:
I guess Scheduled Jobs are Cron Jobs. Correct me if I am wrong 😅
There's no Hindi word for `Cron Jobs`